src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/OldApiScannerInstaller.java

```diff
@@ -93,18 +93,18 @@ public File installOrUpdateScanner() throws BlackDuckIntegrationException {
             }
         }
 
-        File scannerExpansionDirectory = new File(installDirectory, ScannerZipInstaller.BLACK_DUCK_SIGNATURE_SCANNER_INSTALL_DIRECTORY);
+        File scannerExpansionDirectory = new File(installDirectory, OldApiScannerInstaller.BLACK_DUCK_SIGNATURE_SCANNER_INSTALL_DIRECTORY);
```

Comment:
Looking at how some of these constants are used, it seems to me like it would be better to move some of them into the parent installer class (or some other central place). This would help avoid some of the duplication and also make it clearer that some of the properties of the two installers are identical.

Reply:
Good point, thank you for raising this! Moved the BLACK_DUCK_SIGNATURE_SCANNER_INSTALL_DIRECTORY to be in the parent abstract class instead.